LongWarOfTheChosen/Config/XComClassData.ini

```diff
@@ -1737,15 +1740,14 @@ bHasClassMovie=true
 !SoldierRanks=()
 ; squaddie
 +SoldierRanks=(AbilitySlots=((AbilityType=(AbilityName="Shadow", ApplyToWeaponSlot=eInvSlot_Unknown)), \\
```

Comment:
Wait, so you've reworked their vektors to be like strike rifles, but didn't give them squadsight? that seems really strange.

Reply:
Well, you only said the changes would help the weapon work *with* squadsight, and they get squadsight as a potential XCOM row perk to make use of those changes. I'm not ready to give all Reapers squadsight out of the gate.

I do wonder whether Squadsight should be on the weapons themselves, but that would be a larger change that wouldn't be appropriate for 1.0.

Reply:
Then how about putting squadsight somewhere on the main tree?